APMrover2/Log.cpp

```diff
@@ -172,7 +172,7 @@ void Rover::Log_Write_Performance()
 {
     struct log_Performance pkt = {
         LOG_PACKET_HEADER_INIT(LOG_PERFORMANCE_MSG),
-        time_us         : hal.scheduler->micros64(),
+        time_us         : platform::micros64(),
         loop_time       : millis()- perf_mon_timer,
```

Comment:
Vehicles that had their own `millis()` and `micros()` got them replaced with a `using platform::millis` and `using platform::micros`.